version.gradle

```diff
@@ -13,6 +13,7 @@ ext {
     multidexVersion = '2.0.1'
     arcgisVersion = '100.8.0-2747'
     arcgisToolkitVersion = '100.7.0'
+    materialVersion = "1.1.0"
```

Comment:
just for my own learning, what is this new line referring to?

Reply:
we should use single ` ` by the way :)

Reply:
It's being used here @Rachael-E :) https://github.com/Esri/arcgis-runtime-samples-android/pull/783/files#diff-7dba05cc5d8dbf79b99083d8cdbb0e4aR27